unicodetools/src/test/java/org/unicode/text/UCD/TestVersionedSymbolTable.java

```diff
@@ -17,14 +19,17 @@
  * reasonable. If they are broken by changes to property assignments, feel free to update them.
  */
 public class TestVersionedSymbolTable {
+   XSymbolTable old_default;
```

Comment:
optional but cleaner: I would prefer writing UnicodeSet.XSymbolTable here and not importing this nested type. Just like I prefer using `Map.Entry` rather than the mysterious `Entry`.